inspirehep/base/templates/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -54,6 +62,9 @@
                 {% if record.authors %}
                   <div class="authors">
                     {{ render_record_authors(is_brief=true, show_affiliations=false) }}
+                    {% if record.get('earliest_date') %}
```

Comment:
It is like this in Production. Just placed there to have it. I do not think that it fits there correctly.